packages/material-ui/src/Modal/ModalManager.ts

```diff
@@ -69,13 +77,16 @@ function handleContainer(containerInfo, props) {
         el: container,
       });
       // Use computed style, here to get the real padding to add our scrollbar width.
-      container.style['padding-right'] = `${getPaddingRight(container) + scrollbarSize}px`;
+      // TODO: Is there a difference between `padding-right` and `paddingRight`?
+      container.style['padding-right' as 'paddingRight'] = `${
+        getPaddingRight(container) + scrollbarSize
+      }px`;
 
       // .mui-fixed is a global helper.
       fixedNodes = ownerDocument(container).querySelectorAll('.mui-fixed');
```

Comment:
Due to the classname selector the NodeList will never contain `Node` but always at least `Element`. As far as I know in HTML only `HTMLElement` and `SVGElement` extend `Element`.